protocols/exmp-sampling.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+permalink: /protocols/exmp/sampling/
+color: "#593001"        #poop brown
+---
+
+
+#                   EXMP Fecal Sampling Instructions
+
+
+Please read through all of these instructions *BEFORE* you begin collecting a fecal sample.
+
+Trillions of harmless or beneficial microorganisms (tiny living things such as bacteria) live on and within our bodies. During and after birth, our bodies begin to accumulate this diverse community of microorganisms. The composition of this community differs between people and the scientific community is still exploring why this is. There is evidence that our microbial communities are influenced by our parents, our physical environment, what we eat and potentially even our genes. Our resident microbes occupy many body habitats, including the skin and mucosal surfaces, and the gastrointestinal tract.
```

Comment:
I think you should drop this paragraph, since it's not important for the sampling instructions.